### PR TITLE
explain how to install from composer

### PR DESCRIPTION
**Pull request by jokosu10**
add some sentence, how to install package from composer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,16 @@
...
+Add amphp/beanstalk to your composer.json file or you can require amphp/beanstalk with composer. If you are not using Composer, you should be. It's an excellent way to manage dependencies in your PHP application.
+
 ```bash
 composer require amphp/beanstalk
 ```
+
+Then from the command line:
+```bash
+composer install
+```
```
Comment (kelunik) [inline README.md]: `composer require` automatically installed new dependencies. Maybe just change it to the version of https://github.com/amphp/amp#installation with a link to Composer?

Comment (jokosu10) [reply README.md]: you mean replace this scope installation with url  https://github.com/amphp/amp#installation ?

Comment (kelunik) [reply README.md]: Well, the same text, but with this package of course. :+1:

Comment (jokosu10) [reply README.md]: ah i see, oke i can make new pr :)

**Comment by jokosu10** [comment]
please cek my last commit

**Comment by kelunik** [comment]
@jokosu10 Sorry, that's not what I meant. I suggested copying the same format from there, so it's consistent. So something like

```
... Sentence about composer and link to it here ...

\```
composer require amphp/beanstalk
\```
```

(without the `\`)

**Comment by jokosu10** [comment]
ahh i see, as on as possible i make new pull request

**Comment by kelunik** [comment]
I don't think this is really more helpful than the previous content.

**Pull request closed without merging**